### PR TITLE
RavenDB-16264 Don't user batch delete for deleting multiple blobs

### PR DESCRIPTION
**Pull request by garayx**

### CODE REVIEW

File: src/Raven.Server/Documents/PeriodicBackup/Azure/RavenAzureClient.cs [hunk 1]
```diff
@@ -590,6 +590,10 @@ public void DeleteMultipleBlobs(List<string> blobs)
             if (blobs.Count == 0)
                 return;
 
+            DeleteBlobsWithSasToken(blobs);
+
+            //TODO: RavenDB-16264
+            /*
```
Comment (ayende) [inline src/Raven.Server/Documents/PeriodicBackup/Azure/RavenAzureClient.cs:596]: Why do we keep so much dead code?

**Review by ayende — CHANGES_REQUESTED** [review]

File: src/Raven.Server/Documents/PeriodicBackup/Azure/RavenAzureClient.cs (hunk 1 above)
Comment (ppekrol) [reply src/Raven.Server/Documents/PeriodicBackup/Azure/RavenAzureClient.cs:596]: This is not entirely dead code. We want to support multi-delete, but it requires further investigation so for now it is better to go back to a one by one call as I understood from @garayx and @grisha-kotler

**Comment by ppekrol** [comment]
This should go to 4.2?

**Comment by garayx** [comment]
> This should go to 4.2?

yes: https://github.com/ravendb/ravendb/pull/11800

**Pull request closed without merging**